modules/monitoring-modifying-the-retention-time-for-thanos-ruler-metrics-data.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-the-retention-time-for-thanos-ruler-metrics-data_{context}"]
+= Modifying the retention time for Thanos Ruler metrics data
+
+By default, for user-defined projects Thanos Ruler automatically retains metrics data for 24 hours.
```

Comment:
Shouldn't there be a comma after "projects"? 

`By default, for user-defined projects, Thanos Ruler automatically retains metrics...`

Reply:
Yes. Fixed.